src/commons/BrowserUtils.ts

```diff
@@ -906,15 +825,17 @@ export namespace BrowserUtils {
   export function getNumberOfElements(selector: string): number {
     Reporter.debug(`Get number of elements by selector '${selector}'`);
 
-    return $$(selector).length;
+    return findElements(SelectorType.XPATH, selector).length;
   }
 
   /**
-   * Send a sequence of key strokes to the active element
+   * Send a sequence of key strokes to the active element.
+   * You can also use characters like "Left arrow" or "Back space".
+   * WebdriverIO will take care of translating them into unicode characters.
    * it can be single key or an array of keys
    * @param keysToSend key, array of keys or string array (chars) to send
    */
-  export function sendKeys(keysToSend: SpecialKeys | Array<SpecialKeys> | string | Array<string>): void {
+  export function keys(keysToSend: SpecialKeys | Array<SpecialKeys> | string | Array<string>): void {
```

Comment:
Send a sequence of key strokes to the active element. You can also use characters like "Left arrow" or "Back space". WebdriverIO will take care of translating them into unicode characters. You’ll find all supported characters here. To do that, the value has to correspond to a key from the table.

Reply:
what's your point here?

Reply:
that you can remove keysToSend: SpecialKeys | Array<SpecialKeys>